WordPress/WordPressHomeWidgetToday/Views/TodayWidgetView.swift

```diff
@@ -0,0 +1,52 @@
+import SwiftUI
+import WidgetKit
+
+struct TodayWidgetView: View {
+    @Environment(\.widgetFamily) var family: WidgetFamily
+
+    let content: TodayWidgetContent
+
+    @ViewBuilder
+    var body: some View {
+        switch family {
+        case .systemSmall:
+            TodayWidgetSmallView(content: content,
+                                 siteNameTitle: Constants.siteNameTitle,
+                                 viewsTitle: Constants.viewsTitle)
+                .padding(.all, Constants.padding)
+        case .systemMedium:
+            TodayWidgetMediumView(content: content,
+                                  siteNameTitle: Constants.siteNameTitle,
+                                  viewsTitle: Constants.viewsTitle,
+                                  visitorsTitle: Constants.visitorsTitle,
+                                  likesTitle: Constants.likesTitle,
+                                  commentsTitle: Constants.commentsTitle)
+                .padding(.all, Constants.padding)
+        default:
+            Text("View is unavailable")
+        }
+    }
+}
+
+// MARK: - Constants
+extension TodayWidgetView {
+
+    private enum Constants {
+        // Titles
+        static let siteNameTitle: LocalizedStringKey = "Today"
```

Comment:
Can I ask that `siteNameTitle` be renamed? Given the current name, I expect the value to be "Site Name" (i.e. site name title). But it's not, it's actually the widget name. Can it be something like `widgetName` or `widgetTitle`?

Reply:
yup, done!

Reply:
Good catch!